maat-court-data-api/src/main/java/gov/uk/courtdata/link/processor/RepOrderCPInfoProcessor.java

```diff
@@ -21,7 +21,8 @@ public void process(CourtDataDTO courtDataDTO) {
         final CaseDetails caseDetails = courtDataDTO.getCaseDetails();
         Optional<RepOrderCPDataEntity> repOrderEntity = repOrderDataRepository.findByrepOrderId(caseDetails.getMaatId());
 
-        RepOrderCPDataEntity repOrder = repOrderEntity.get();
+        RepOrderCPDataEntity repOrder = repOrderEntity.orElse(null);
+        assert repOrder != null;
```

Comment:
I personally think we should avoid 'assert'.

Reply:
Done!!